src/main/resources/i18n/messages.properties

```diff
@@ -98,24 +98,19 @@ ranked1v1.rankingFormat=#{0,number,#}
 ranked1v1.notification.title=1v1 ranked match available
 ranked1v1.notification.message=Click here to open the 1v1 tab
 teammatchmaking.leaveParty=Leave party
-teammatchmaking.title=Team matchmaking
 teammatchmaking.invitePlayer=Invite player
 teammatchmaking.invite=Invite
-teammatchmaking.availableQueues=Available queues
-teammatchmaking.playersInQueue=In Queue: {0,number,#}
-teammatchmaking.teamSize=Team size: {0,number,#}
-teammatchmaking.joinQueue=Join
-teammatchmaking.leaveQueue=Leave
+teammatchmaking.playersInQueue=IN QUEUE: {0,number,#}
 teammatchmaking.queuePopTimer={0,number,0}:{1,number,00}
-teammatchmaking.chat=Chat / Map pool?
-teammatchmaking.statistics=Statistics / Chat?
-teammatchmaking.rating=Placement (1v1)
-teammatchmaking.gameCount=Games: {0,number,#}
+teammatchmaking.rating=RATING: {0,number,#}
+teammatchmaking.1v1Rating=1V1 RATING: {0,number,#}
+teammatchmaking.teamRating=TEAMGAME RATING: {0,number,#}
+teammatchmaking.gameCount=GAMES PLAYED: {0,number,#}
```

Comment:
Caps breaks with the UI style of the rest of the client

Reply:
A lot of this breaks with the ui style of the client. It needs the new font to look good, but we cant have everything be small letters that looks weird for important texts.